electron_app/src/Account.js

```diff
@@ -77,6 +78,9 @@ class Account {
   get autoBackupPath() {
     return this.activeAccount.autoBackupPath;
   }
+  get loggedAccounts() {
+    return this._loggedAccounts;
```

Comment:
When updating accounts, I look it in active or inactive, a third variable will be harder to keep track 
```suggestion
    return this.activeAccount + this.inactiveAccounts;
```